include/core/mir/geometry/size.h

```diff
@@ -65,6 +65,11 @@ inline constexpr Size operator*(Size const& size, Scalar scale)
     return scale*size;
 }
 
+inline constexpr Size operator+(Size const& lhs, Size const& rhs)
+{
+    return Size{lhs.width + rhs.width, lhs.height + rhs.height};
```

Comment:
The other operators introduced here are arguable (while adding widths has a clear semantic, I'm not sure it is useful).

Howeverm _this_ one is wrong. (As discussed in #1064)

Reply:
Removed